hi-ml-cpath/src/health_cpath/datasets/panda_dataset.py

```diff
@@ -42,9 +42,10 @@ def __init__(self,
                  dataset_csv: Optional[Union[str, Path]] = None,
                  dataset_df: Optional[pd.DataFrame] = None,
                  label_column: str = "isup_grade",
-                 n_classes: int = 6) -> None:
+                 n_classes: int = 6,
+                 **kwargs: Any) -> None:
```

Comment:
It's mainly because we were adding extra params in the base class recently and forgot to propagate them to child classes, using **kwargs makes it easier to extend base classes args. Or should I add all params in base class?  (Note we should keep in mind updating subclasses in the future)

Reply:
I guess it depends how many params we are talking about adding...can discuss offline